pyproject.toml

```diff
@@ -1,3 +1,69 @@
+[project]
+name = "nle"
+description = "The NetHack Learning Environment (NLE): a reinforcement learning environment based on NetHack"
+readme = "README.md"
+authors = [ { name = "The NLE Dev Team" } ]
+requires-python = ">=3.10"
+dynamic = ["classifiers", "version"]
+dependencies = [
+  "pybind11>=2.2",
+  "numpy>=1.16",
+  "gymnasium==1.2.0"
+]
+
+[project.license]
+file = "LICENSE"
+
+[project.urls]
+Homepage = "https://github.com/NetHack-LE/nle"
+
+[build-system]
+requires = ["setuptools", "pybind11>=2.2"]
```

Comment:
Add `"cmake"` here, and any other package that is required to be present during build time.

It also tends to be a good idea to pin version here since unfortunately uv does not yet support locking of build deps: https://github.com/astral-sh/uv/issues/5190